api/models/cartDao.js

```diff
@@ -0,0 +1,90 @@
+const { myDataSource } = require("./myDataSource");
+
+
+const getCart = async (userId) => {
+  const cartList = await myDataSource.query(
+    `
+  SELECT
+    c.id AS cId,
```

Comment:
컬럼명을 AS로 치환할 때 더 명확하게 해주세요. cId -> cartId
다른 컬럼들도 전부 확인 후 변경해 주세요.
```suggestion
    c.id AS cartId,
```